src/types/actions.js

```diff
@@ -20,7 +20,12 @@ type BatchAction = {
     payload: Array<GenericAction>;
 }
 
-type Action = GenericAction | Thunk | BatchAction
+export type PostAction = {
+    ...GenericAction,
+    postId?: string
```

Comment:
This looks like an exception only used in the `RECEIVED_FILES_FOR_POST` action. Maybe I would prefer a explicit `ReceivedFilesForPostAction` with the definition similar to:

```javascript
{|
    type:FileTypes.RECEIVED_FILES_FOR_POST,
    postId: string,
    data: Array<FileInfo>,
|}
```